schemas/com.snowplowanalytics.amp/amp_id/jsonschema/1-0-0

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self": {
+    "vendor": "com.snowplowanalytics.amp",
+    "name": "amp_id",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "description": "Schema for AMP user identification",
+  "properties": {
+    "ampClientId": {
+      "type": "string",
+      "description": "AMP client ID - unique to AMP user",
+      "maxLength": 128
+    },
+    "domainUserId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "Snowplow domain userid - tracked if available",
+      "maxLength": 128
+    },
+    "userId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "User-defined userid - tracked if available",
+      "maxLength": 255
+    }
+  },
+  "additionalProperties": false,
```

Comment:
Are we sure AMP won't add more properties on a whim? Or is this data coming from our tracker, meaning we control the fields?

Reply:
Yes. We control it.